### ISSUE TITLE
PHPCI not showing phpunit result

### ISSUE THREAD

**Issue by simonchillier** [Block8/PHPCI#1330]
Hi,

when I am running a build in my PHPCI environment, everything work fine... but just a small problem with the php_unit plugins. Is not showing the tests results

**Here the build log** 
Cloning into '/Users/xxxx/Projects/phpci/PHPCI/build/138_f2255'...

2ba824bc978baf834ceaee74833629039d020cb8
Working copy created: /
**RUNNING PLUGIN: php_unit
PLUGIN: SUCCESS**
RUNNING PLUGIN: php_code_sniffer
PLUGIN: SUCCESS
BUILD SUCCESS
RUNNING PLUGIN: email
1 emails sent
0 emails failed to send
PLUGIN: SUCCESS
Removing Build.

**Here my phpci.yml**
build_settings:
    verbose: true
    clone_depth: 2
    ignore:
        - "3rdparty"
        - "certs"
        - "SQL_changes"
        - "assets"
    mysql:
        host: "xxx"
        user: "xxx"
        pass: "xxxx"

test:
   
    php_unit:
        config:
            - "phpunit.xml"
        coverage: "build/coverage"
        #path: "phpunit/"
#        directory:
#            - "phpunit/"
#        run_from: "phpunit/"
  
    php_code_sniffer:
        path: "certs"
        standard: "PSR1,PSR2"
        allowed_warnings: 999
        allowed_errors: 999
        
complete:
    email:
        default_mailto_address: xxx@xxx.com

**Here my phpunit.xml**
<?xml version="1.0" encoding="UTF-8"?>
<phpunit>
    <testsuites>
        <testsuite name="Test suite">
            <directory>/Users/xxx/Projects/xxx/phpunit/</directory>
        </testsuite>
    </testsuites>
		<filter>
        <whitelist processUncoveredFilesFromWhitelist="true">
            <directory>/Users/xxx/Projects/xxx/phpunit</directory>
        </whitelist>
    </filter>
		<logging>
			<log type="json" target="/Users/xxxx/Projects/xxx/logs/logfile.json"/>
	</logging>
</phpunit> 

**Here my test**

final class LoginTest extends TestCase
{
	
    public function testLoginEnergyCloud2()
    {
	$this->assertEquals(
            [1, 2, 33, 4, 5, 6], 
            [1, 2, 3, 4, 5, 6]
        );
		
	}
}

The test should be return a Failure result. 
When I am run manually the test, with this command : phpunit  --configuration /Users/xxx/Projects/xxx/phpunit.xml is working correctly and i receive a test failure result.

I did something wrong but I don't know why. Is a log file for PHPCI while my logging in my phpunit config file is not working properly (the file is empty)

Thanks
Simon

**Comment by Ooypunk** [comment]
Have you checked the "Information" tab?

Logfile being empty: curious. Could it be that PHPCI intercepts the output?

**Comment by simonchillier** [comment]
yes I checked the information tab. I will continue my research and keep you in touch.

Thanks
